test/src/main/java/org/apache/accumulo/test/shell/ShellServerIT.java

```diff
@@ -474,35 +512,23 @@ public void setIterOptionPrompt() throws Exception {
       // Name on the CLI should override OptionDescriber (or user input name, in this case)
       ts.exec("setiter -scan -class " + COLUMN_FAMILY_COUNTER_ITERATOR + " -p 30 -name cfcounter",
           true);
-      expectedKey = "table.iterator.scan.cfcounter";
-      expectedValue = "30," + COLUMN_FAMILY_COUNTER_ITERATOR;
-      checkTableForProperty(tops, tableName3, expectedKey, expectedValue);
-      expectedKey = "table.iterator.scan.cfcounter.opt.name1";
-      expectedValue = "value1.1,value1.2,value1.3";
-      checkTableForProperty(tops, tableName3, expectedKey, expectedValue);
-      expectedKey = "table.iterator.scan.cfcounter.opt.name2";
-      expectedValue = "value2";
-      checkTableForProperty(tops, tableName3, expectedKey, expectedValue);
 
-      ts.exec("deletetable " + tableName3, true);
+      assertTrue(Wait.waitFor(
+          () -> client.tableOperations().getConfiguration(tableName3)
+              .get("table.iterator.scan.cfcounter").equals("30," + COLUMN_FAMILY_COUNTER_ITERATOR),
+          5000, 500));
 
-    }
-  }
+      assertTrue(Wait.waitFor(
+          () -> client.tableOperations().getConfiguration(tableName3)
+              .get("table.iterator.scan.cfcounter.opt.name1").equals("value1.1,value1.2,value1.3"),
+          5000, 500));
 
-  protected void checkTableForProperty(TableOperations tops, String tableName, String expectedKey,
```

Comment:
An alternative to removing this method could be to move the `assertTrue(Wait.waitFor())` blocks that were added in this PR into this method. Not sure if it would be better but if that were to happen then I think a lot of this code would not need to be changed and the logic could be reused.

Reply:
It seemed better to favor explicitly calling `Wait.waitFor()` because that made it clear there in the test what was happening - wrapping it in checkForTableProperty() was unique to this test and `Wait.waitFor()` more common elsewhere - but either way is fine if reducing the number of changes is more important for this change for this test.

Reply:
Addressed in 455558d458